doc/engineering-reference/src/integrated-solution-manager/zone-air-mass-flow-conservation.tex

```diff
@@ -175,7 +229,7 @@ \subsection{Infiltration Flow Rate Adjustments}\label{infiltration-flow-rate-adj
 If a zone is included in the infiltration adjustment, the infiltration air mass flow rate required to balance the zone is determined as follows:
 
 \begin{equation}
-{\dot m_{Inf-required}} = MAX\left( {0.0,\,{{\dot m}_{XS}} + {{\dot m}_{EX}} + {{\dot m}_{R}} - {{\dot m}_S}} \right)
+{\dot m_{Inf-required}} = MAX\left( {0.0,\,{{\dot m}_{XS}} + {{\dot m}_{EX}} + {{\dot m}_{R}} - {{\dot m}_S}  - {\dot m_{XR,\,new}}} \right)
```

Comment:
Added a missing terms in the air air infiltration adjusting or adding calculation option. This correction allows infiltration treatment for receiving zone as well.